PicaSWB.js

```diff
@@ -238,14 +250,24 @@ function doExport() {
 			if (item.publisher) { publicationStatement +=  "$n" + item.publisher; }
 			writeLine("4030", publicationStatement);
 		}
-		
-		//4070 $v Bandzählung $j Jahr $h Heftnummer $p Seitenzahl
+			//4070 $v Bandzählung $j Jahr $h Heftnummer $p Seitenzahl|case:magazineArticle 
 		if (item.itemType == "journalArticle") {
 			var volumeyearissuepage = "";
 			if (item.volume) { volumeyearissuepage += "$v" + item.volume; }
 			if (date.year !== undefined) { volumeyearissuepage +=  "$j" + date.year; }
 			if (item.issue) { volumeyearissuepage += "$h" + item.issue; }
 			if (item.pages) { volumeyearissuepage += "$p" + item.pages; }
+			
+			writeLine("4070", volumeyearissuepage);
+		}
+		//4070 $v Bandzählung $j Jahr $h Heftnummer $p Seitenzahl
+		if (item.itemType == "magazineArticle") {
```

Comment:
Die Informationen sind doch bei Zeitschriftenartikel und "magazinArticle" gleich nur die if-Bedingung unterscheidet sich. Richtig? Dann kann man einfach ein Oder in die Bedingung machen, d.h.
```
if (item.itemType == "journalArticle" || item.itemType == "magazineArticle") {
```